### ISSUE TITLE
Increase test speed

### ISSUE THREAD

**Issue by ahamelers** [datadryad/dryad-product-roadmap#3724]
Our rspec test suite takes an incredibly long time to run—with changes for the redesigned submission system, this increases to 30 minutes 🤯 

One of the reasons is all of the tests have :before_each tasks that are sometimes, themselves, very time consuming. We could change this in many cases to do :before_all and possibly only perform these time-consuming sequences once per test set, but this is currently not possible, and the database is wiped after each test (https://github.com/datadryad/dryad-app/blob/main/spec/support/database_cleaner.rb)

This could be changed but it would need to be done carefully, as all the tests now take for granted that at the start of each they're working with a blank, fresh database.

Any other or alternative ideas for speeding up the rspec tests would also be greatly appreciated!

**Comment by alinvetian** [comment]
rspec resets contexts per `IT`, also if you invoke subject on each IT you will be executing the method once per each.
Usually, a context would need only one it. That might have multiple assertions.

**Comment by ahamelers** [comment]
@alinvetian I have already tried to modify the tests to do some of what you're suggesting and they are still taking a million years; I could die, turn to dust, and float away waiting for these tests to complete. Please help! We also have https://github.com/datadryad/dryad-product-roadmap/issues/1812 open

**Comment by alinvetian** [comment]
Yes, parallelization is the way to go with minimum coding effort.

**Comment by ryscher** [comment]
Please ensure this works with the changes in the branch for the submission system changes.

**Comment by alinvetian** [comment]
Also fixing this flaky one https://github.com/datadryad/dryad-app/blob/main/spec/features/stash_datacite/new_collection_spec.rb#L32
